Resources/Textures/DeltaV/Structures/Wallmounts/intercom.rsi/meta.json

```diff
@@ -1,13 +1,37 @@
 {
     "version": 1,
     "license": "CC-BY-SA-3.0",
-    "copyright": "Taken from /tg/station at https://github.com/tgstation/tgstation/commit/6bfe3b2e4fcbcdac9159dc4f0327a82ddf05ba7bi. Modified by Radezolid for random intercom spawner",
+    "copyright": "Taken from /tg/station at https://github.com/tgstation/tgstation/commit/6bfe3b2e4fcbcdac9159dc4f0327a82ddf05ba7bi, modified by JustAnOrage, Modified by Radezolid for random intercom spawner",
     "size": {
         "x": 32,
         "y": 32
     },
     "states": [
         {
+            "name": "base",
+            "directions": 4
+        },
+        {
+            "name": "broadcasting",
+            "directions": 4
+        },
+        {
+            "name": "build",
+            "directions": 4
+        },
+        {
+            "name": "speaker",
+            "directions": 4
+        },
+        {
+            "name": "unshaded",
+            "directions": 4
+        },
+        {
+            "name": "panel",
+            "directions": 4
+        },
+		{
```

Comment:
untroll indent on this line

Reply:
I don't see any indents on my end...?

Reply:
mixing tabs and spaces

Reply:
Where though? 
![image](https://github.com/user-attachments/assets/75b83cd8-31e5-4ac3-a7ab-af23790eb4cc)
![image](https://github.com/user-attachments/assets/43c74b01-177d-4a58-9d3f-ee386b636e5c)

Reply:
your ide cant render it so ill fix tomorrow ig